spec/features/admin/page_destroy_feature_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+module Alchemy
+  describe "Page destroy feature", js: true do
+    before { authorize_user(:as_admin) }
+
+    context 'destroying a content page' do
+      let!(:content_page) { create(:alchemy_page) }
+
+      it "redirects to page tree" do
```

Comment:
The expectation claims a redirect which is actually not tested in this test. Because: If you would not redirect in the controller but only reload part of the page the test would be still green because `page.current_path` is the same, right?

Besides this the expectation to this feature should be different in my opinion: I'd prefer to test that the page tree is rendered without the destroyed page. This would test the actual destroy action and also covers the rendering of the page tree after the deletion.